### DISCUSSION TITLE
Drive Download with Flask

### DISCUSSION THREAD

**Discussion by lthon09** [deta/cloud-docs#331] [Help] [answered]
The webserver is written with Flask, Drive's Python `.get()` returns the file's content IIRC. I wanna return an attachment for the user to download it. I'm wondering if this is possible with Flask, since I only know about returning files from the directory only. Thanks!

**Comment by jnsougata** [comment, marked as answer]
https://tedboy.github.io/flask/generated/flask.send_file.html#flask-send-file
this might help if you just save the file temporarily then send.... 
idk if flask has support for sending file content directly

**Reply by lthon09** [reply]
ive thought about that, but thats just really ineffective. but maybe thats my only way besides shipping it with the micro

**Reply by lthon09** [reply]
would it work if i write it into memory?

**Reply by lthon09** [reply]
maybe this is the way

**Reply by jnsougata** [reply]
yes should work... fastpai has better implementation of file responses

**Reply by lthon09** [reply]
alright, writing the file content into memory and sending it seems like the best way known for flask
